src/tools/dart-run.md

```diff
@@ -129,13 +129,30 @@ $ dart run foo arg1 arg2
 ## Debugging
 
 To enable debugging, pass one or more debugging options.
-Here's an example of enabling [`assert` statements][assert]:
+
+To enable [`assert` statements][assert],
+specify the `--enable-asserts` flag:
 
 ```terminal
 $ dart run --enable-asserts tool/debug.dart
 ```
 
-See `dart run --help` for details.
+To enable debugging through [Dart DevTools](/tools/dart-devtools),
+a set of debugging and performance tools,
+specify the `--observe` flag.
+Optionally add `--pause-isolates-on-start`
+to break execution at the start of the specified script:
+
+```terminal
+$ dart run --pause-isolates-on-start --observe tool/debug.dart
+```
+
```

Comment:
**issue:** The resulting link is not shown here. It would help understanding to include it.


```terminal
The Dart VM service is listening on http://127.0.0.1:8181/afZySiNbDPg=/
/?uri=ws%3A%2F%2F127.0.0.1%3A8181%2FafZySiNbDPg
```